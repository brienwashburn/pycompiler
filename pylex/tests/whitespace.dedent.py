def f(x):
  	if x:
  		 print("hello world\n")