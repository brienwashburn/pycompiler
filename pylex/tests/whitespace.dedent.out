(KEYWORD def)
(ID "f")
(PUNCT "(")
(ID "x")
(PUNCT ")")
(PUNCT ":")
(NEWLINE)
(INDENT)
(KEYWORD if)
(ID "x")
(PUNCT ":")
(NEWLINE)
(INDENT)
(ID "print")
(PUNCT "(")
(LIT "hello world\n")
(PUNCT ")")
(DEDENT)
(DEDENT)
(ENDMARKER)
